.github/workflows/public_docker_images.yml

```diff
@@ -0,0 +1,89 @@
+# https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages
+#
+name: Docker image on ghcr.io
+
+on:
+  push:
+    tags:
+      - 'v*'
+  pull_request:
+    branches: main
+  schedule:
+    - cron: '0 2 29 * *'  # At 02:00 on day-of-month 29
```

Comment:
@felixT2K I changed the cron so it should be triggered tomorrow morning. We'll see if it works or not

Reply:
alright 👍🏼